gradleops/AndroidSeed/app/src/main/AndroidManifest.xml

```diff
@@ -86,6 +86,7 @@
 				<data android:scheme="package"/>
 			</intent-filter>
 		</receiver>
+	<uses-library android:name="org.apache.http.legacy" android:required="false" />
```

Comment:
what does this change do ?

Reply:
googled it :) It is for implementations using the old Http method.

Reply:
yes